.github/actions/install-chrome/action.yml

```diff
@@ -5,15 +5,20 @@ runs:
   using: 'composite'
   steps:
     - id: install-chrome
-      name: Install Chrome
+      name: Install Chrome (retries on error)
+      uses: nick-fields/retry@v3
```

Comment:
Checklist for adding dependency:

* [x] **Actively maintained based on recent commit history:** Updates are infrequent, but there's not much to change about this action. https://github.com/nick-fields/retry/commits/master/
* Code changes are secured by one of the following:
  * [x] **Only a small group of people have commit access:** Single maintainer
  * [ ] **Code review is enforced:**
* [x] **No binary artifacts:** per scorecard
* [x] **Any unfixed vulnerabilities reported by GitHub are minor or not relevant:**
  * GHSA-grv7-fg5c-xmjg: Memory DOS upon malicious input. Not a problem because (a) we control input to the action and (b) we don't use this dependency in production
  * GHSA-3xgq-45jj-v275: Regex DOS upon malicious input. Not a problem because (a) we control input to the action and (b) we don't use this dependency in production
  * GHSA-78xj-cgh5-2h22: IP addresses mis-classified as public. OK because we don't rely on this package for IP address processing
  * GHSA-2p57-rm9w-gvfp: IP addresses mis-classified as public. OK because we don't rely on this package for IP address processing
  * GHSA-952p-6rrq-rcjv: Regex DOS upon malicious input. Not a problem because (a) we control input to the action and (b) we don't use this dependency in production
  * GHSA-c2qf-rxjj-qqgw: Regex DOS upon malicious input. Not a problem because (a) we control input to the action and (b) we don't use this dependency in production
  * GHSA-f5x3-32g6-xq36: Un-compressing DOS upon malicious input. Not a problem because (a) we control input to the action and (b) we don't use this dependency in production
  * GHSA-3787-6prv-h9w3: Failure to clear authorization headers upon redirect. OK because we don't use this package for any redirection.
  * GHSA-9qxr-qj54-h672: Attackers who can control input can bypass integrity checks. OK because we control inputs, which are fixed.
  * GHSA-m4v8-wqvr-p9f7: Failure to clear authorization headers upon redirect. OK because we don't use this package for any redirection.
* [x] **Compatible license:** https://github.com/nick-fields/retry#MIT-1-ov-file
* [ ] **Reputable maintainer(s):** Consider whether they are well-known figures or connected to well-known figures. For example, do they work for a known company? Make sure to verify any connections, e.g. by membership in GitHub organizations or checking company directories. This is very light vetting and is not intended to show that the maintainer is trustworthy. The goal is to convince ourselves that the maintainer has enough "reputation signals" that it'd be unrealistic for a bad actor to build up these signals for malicious purposes.
* [x] **Established project:** Running since at least 2020 https://github.com/nick-fields/retry/issues/4

Looks fine, except that I wasn't able to find much on the maintainer. For an action this simple, should we just implement this ourselves?

Reply:
Eh, I think it's fine to use this one. It came up on various searches I was doing for retry functionality, and I believe we already use an older version of the same action (at least, we have allowlisted that previously). 

I'd rather leave it as a third-party dep for now, I think, if there are no significant concerns -- less code for us to maintain, and we can always bring it in-house if we need to customize it.

Reply:
Just for posterity: I'm aware of another retry action, but that one seems to have fewer usages/interest AFAICT: https://github.com/marketplace/actions/retry-action (corresponds to https://github.com/Wandalen/wretry.action)